File: BlackBerry/AlertMsgs/LevelAlert.java
package com.mlhsoftware.Blurts;

import org.json.me.*;


class LevelAlert extends AlertBase 
{
  // Tag Names
  private static final String ALERT_NAME  = "Level";
  private static final String KEY_BATTERY = "Battery";
  private static final String KEY_SIGNAL  = "Signal";


  public LevelAlert( String string ) throws JSONException
  {
    super( string );
  }

  public LevelAlert()
  {
    super( TYPE_LEVEL, ALERT_NAME );
  }



}

File: Desktop/Domain/AlertMsgs/LevelAlert.cs
using System;
using System.Runtime.InteropServices;
using Blurts.Domain;

namespace Blurts
{
  [ComVisible(true)]
  public class LevelAlert : AlertBase
  {
    // Tag Names
  private static String ALERT_NAME  = "Level";


   
    public LevelAlert(String s)
      : base(s)
    {
    }

    public LevelAlert()
      : base(DisplayItemType.LEVEL, ALERT_NAME)
    {
    }

    public override Boolean ProcessMessage()
    {
      this.DisplayAlert = false;
      this.PlaySound = false;
      runScript( "OnLevelAlert" );
      return true;
    }
  }
}
